internal/core/graphics/brush.rs

```diff
@@ -53,6 +53,54 @@ impl Brush {
         }
     }
 
+    /// Returns the red channel of a color
+    ///
+    /// ```
+    /// # use i_slint_core::graphics::*;
+    /// assert_eq!(Brush::default().red(), 0);
+    /// assert_eq!(Brush::SolidColor(Color::from_argb_u8(0, 255, 128, 140)).red(), 255);
+    /// assert_eq!(!Brush::SolidColor(Color::from_argb_u8(25, 128, 140, 210)).red(), 128);
+    /// ```
+    pub fn red(&self) -> u8 {
```

Comment:
Do we really want to add these function in our Rust API (to brush and color)
The way to do get the red value in rust would be: `brush.color().to_argb_u8().red`
A brush is not necessarily a color, so the `color()` makes it explicit there is a conversion.  Also the _u8 or _f32 makes it clear what this does.

We don't need to have that to expose it in .slint. The code generator can call this as well.

Reply:
Yeah this was a mistake on my part through not fully understanding the difference between  color and brush